stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/extensions/ChatClient.kt

```diff
@@ -198,7 +199,7 @@ public fun ChatClient.downloadAttachment(context: Context, attachment: Attachmen
             val logger = StreamLog.getLogger("Chat:DownloadAttachment")
             val downloadManager = context.getSystemService(Context.DOWNLOAD_SERVICE) as DownloadManager
             val url = attachment.assetUrl ?: attachment.imageUrl
-            val subPath = attachment.name ?: attachment.title
+            val subPath = attachment.name ?: attachment.title ?: attachment.getAttachmentFallbackName() ?: "attachment"
```

Comment:
Can we use a `Date()` or `Timestamp` as the last default one?
It will prevent we override the attachments if they don't have a proper file name.

Reply:
Switched to using a dated string